x/examples/outline-connectivity-app/shared_frontend/pages/connectivity_test/index.ts

```diff
@@ -0,0 +1,802 @@
+      font-family: var(--font-sans-serif);
+      font-size: var(--size-font-small);
+    }
+
+    .footer-selector {
+      background: var(--color-text-muted);
+      border-radius: var(--size-corner-radius);
+      color: var(--color-background-muted);
+      cursor: pointer;
+      font-family: var(--font-sans-serif);
+      text-align-last: center;
+      font-size: var(--size-font-small);
+      padding: var(--size-gap-inner) var(--size-gap-narrow);
+    }
+
+    .footer-selector:focus,
+    .footer-selector:hover {
+      background: var(--color-background-highlight);
+      color: var(--color-text-highlight);
+    }
+
+    @media (prefers-reduced-motion: reduce) {
+      .field-input-checkbox,
+      .field-input-checkbox::after {
+        transition: none;
+      }
+    }
+
+    @media (prefers-contrast: more) {
+      :host {
+        --size-border: 2px;
+      }
+
+      .field-group {
+        border: var(--size-border) solid var(--color-text);
+        background: var(--color-background);
+      }
+
+      .field-header-info {
+        opacity: 1;
+      }
+
+      .field-input-checkbox {
+        border-color: var(--color-text);
+      }
+
+      .field-input-checkbox:checked {
+        background: var(--color-background);
+        border-color: var(--color-text);
+      }
+
+      .results-header {
+        border: var(--size-border) solid var(--color-text);
+        background: var(--color-background);
+      }
+
+      .results-header-close {
+        opacity: 1;
+      }
+
+      .results-list-item,
+      .results-list-item:last-child {
+        background: var(--color-background);
+        border-width: 0 var(--size-border) var(--size-border) var(--size-border);
+        border-bottom: var(--size-border) solid var(--color-text);
+      }
+
+      .results-list-item-data-key,
+      .results-list-item-data-value {
+        opacity: 1;
+        color: var(--color-text);
+      }
+
+      .field-input,
+      .field-input-textarea {
+        background: var(--color-background);
+      }
+
+      .footer {
+        border-top: var(--size-border) solid var(--color-text);
+        background: var(--color-background);
+      }
+
+      .footer-separator {
+        color: var(--color-text);
+      }
+
+      .footer-selector-label {
+        color: var(--color-text);
+      }
+
+      .footer-selector {
+        background: var(--color-text);
+        color: var(--color-background);
+      }
+    }
+
+    
+    @media only screen and (max-width: 480px) {
+      /* if mobile */
+      .results-list-item-data {
+        display: flex;
+        flex-direction: column;
+        gap: var(--size-gap-inner);
+      }
+
+      .footer {
+        display: none;
+      }
+
+      .field-header-info {
+        display: none;
+      }
+
+      /* if ios */
+      .header {
+        padding-top: 3.7rem;
+      }
+    }
+  `;
+
+  render() {
+    return html`<main dir="${this.locale === "fa-IR" ? "rtl" : "ltr"}">
```

Comment:
This is not quite maintainable. Consider keeping a [centralized list of supported languages](https://github.com/Jigsaw-Code/outline-client/blob/0cae8d0fb1ac2706ba25dfd9ea93b657ca6db45b/src/www/ui_components/app-root.js#L442), or perhaps try using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/getTextInfo

Reply:
Yes